From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

4c74356b41 commented:
also you would want to replace automation cmdlets link on the right with the proper one
https://msdn.microsoft.com/en-us/library/mt244122.aspx

![untitled](https://cloud.githubusercontent.com/assets/16194643/19922185/98312a68-a0f2-11e6-96ee-2d5f6d8a5bce.png)

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @mgoedtel, has been notified to review your proposed change.

MGoedtel commented:
@4c74356b41 Thank you for your suggested changes to this article.  Upon review of your proposed changes, they are incomplete where the example PowerShell script would not complete successfully.  Additionally, the links to the ARM cmdlets were not included with your suggested change.  I've updated the article accordingly and updated/validated the script logic.  I'll submit a PR now and will also submit another PR to update the TOC to reference the Automation ARM cmdlets.   I cannot approve this PR.

4c74356b41 commented:
how can i update the navigation menu on the left? i mean where do i supply the pr? also yeah, i forgot the resourcegroup in there, sorry

MGoedtel commented:
@4c74356b41 - You can't, its not a task that can be performed from the public repo.  I identified another section in the article that needs re-working, so I'm addressing that now (specifically - Viewing job status from the portal.  It's referencing the old portal not the new portal...)

4c74356b41 commented:
what did you mean with the link to arm commadlets then? I've updated the links in my pr...

MGoedtel commented:
@4c74356b41 - I was referring to the link on MSDN for each of the cmdlets.  Originally you had updated the reference to the ARM cmdlet, but not the link to the MSDN topic.  It was addressed in my PR and today/tomorrow I am releasing another update to this article to address outdated details/information in other sections of the article.

ktoliver commented:
@4c74356b41 - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/automation/automation-runbook-execution](https://docs.microsoft.com/en-us/azure/automation/automation-runbook-execution). Thanks for your understanding.

Commit:
Adding large content hadling with chunking documentation

Changing some formatting stuff

Changing some formatting stuff x2

Changing some formatting stuff x3

Changing some wording

Changing some wording x2

rebasing

tdykstra committed:
new webjobs sdk documentation

Commit:
tmp

Commit:
Add how to use SDK doc and modify toc

Commit:
add tool reference

Commit:
addressing Pierre's comment

Commit:
addressing nicole's feadback

Commit:
update 1

SaeedAkhter-MSFT committed:
388665 review auth scenarios

Commit:
modify toc

Commit:
Merge https://github.com/MicrosoftDocs/azure-docs-pr into ios-doc

Commit:
add how to guide for mobile devices

Commit:
update link

Commit:
remove reference

Commit:
Merge https://github.com/MicrosoftDocs/azure-docs-pr into ios-doc

Commit:
update reference:

Commit:
resolve conflict

Commit:
clean up

Commit:
addressing comments

Commit:
addressing feedback

Commit:
update link for sample repo

Commit:
Changes

Commit:
Fixing some wording

Commit:
fix styles

Commit:
change author name

Commit:
change title

Commit:
Rewording

gled4er committed:
Adding new page for Custom orchestration status.

gled4er committed:
Adding custom orchestration status to the HTTP API page.

ramical committed:
Update sample to use AcquireTokenAsync

Update sample to use AcquireTokenAsync

mmacy committed:
[ACR] new: ACR Build overview

gled4er committed:
Addressing feedback from review.

v-viinde committed:
USER STORY 1245548 SaaS App Tutorial: Montage Online

mmacy committed:
[ACR] minor wording

anirudhcavale committed:
Updating available metrics

Updating metric availability for MD, PowerBI, MySQL, and PostgreSQL

anirudhcavale committed:
Update MySQL + PostgreSQL

anirudhcavale committed:
Update MySQL

Commit:
Updated TOC to match MVC:
 Created Landing page with tutorial icons.
 Added SDKs per API Reference Guidance.
 Added common Resources items.
Split samples-dotnet in 2:
 REST samples now in samples-rest.
Deleted sdk-and-samples as redundant.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ContentModerator

Commit:
Updating from PR comments x2

Commit:
index: added missing slash to url's.
quick-start: removed broken link to sdk-and-samples.md.
toc: added missing 's' to FAQ.
index and samples-rest: changed author to easyj2j.

Commit:
Add back.

v-viinde committed:
updated

v-viinde committed:
updated

v-viinde committed:
updated

v-nagta committed:
User Story 1246719: SaaS App Tutorial: Jamf Pro SAML Connector

v-nagta committed:
User Story 1246722: SaaS App Tutorial: LaunchDarkly

v-viinde committed:
updated

v-nagta committed:
uodate

Commit:
Updated to match MVC.

toc: added Landing page.
Made Overview a dropdown with 1 item: About ...
Made SDKs a dropdown with new SDKs per API Reference Guidance.
Added stock Resources items.

Created new Landing page with Tutorial icons.

home: Changed "Overview" to "About Custom Vision Service".
Fixed some Acrolinx warnings.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CustomVision

Commit:
Removed region constraints

Commit:
redirection: deleted Custom-Vision-Service redirection from index to home.
index: removed "home" from Custom-Vision-Service link.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CustomVision

mattbriggs committed:
Update profile index

mmacy committed:
[ACR] TOC and titles

mattbriggs committed:
clearing up issue

Commit:
Rewording as a how to guide

mattbriggs committed:
update note formatting

mmacy committed:
[ACR] section updates

v-deken committed:
Updated landing pages for Autosuggest, Custom Search, Entity Search, and Image Search

Commit:
Couple more rewording

v-deken committed:
Updated landing pages for Autosuggest, Custom Search, Entity Search, and Image Search

mmacy committed:
[ACR] quick build/scc commit sections

Commit:
Adding MB hyphen

Commit:
Removing the 30-MB hyphen when not noun

v-deken committed:
Added converted landing page for Custom Search to TOC

v-deken committed:
Added cards to Reference sections for SDK

v-deken committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into LandingPage-RestOfSearch

v-deken committed:
Added cards to Reference sections for SDK

Commit:
Made changes from cjgronlund & JennieHubbard <notifications@github.com>.

Deleted landing page (index).
Overview: Changed "About " to "What is ".

toc: deleted index.
Renamed Overview sub-item.
Renamed Quickstarts to "Try Content Moderator" and moved to "How to".
Renamed Tutorials to "First Content Moderator apps" and moved to "How to".

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ContentModerator

Commit:
Made changes from cjgronlund & JennieHubbard <notifications@github.com>.

redirection.json: aded Custom-Vision-Service redirection from index to home.
index: added "home" to Custom-Vision-Service link.

Deleted landing page (index).
Overview (home): Changed "About " to "What is ".

toc: deleted index.
Renamed Overview sub-item.
Changed "How-to guides" to "How-to".
Renamed Quickstarts to "Start using Custom Vision Service" and moved to "How to".
Renamed Tutorials to "First Content Moderator apps" and moved to "How to".

C# and Python tutorials: changed "Computer Vision" to "Custom Vision Service".
Fixed some Acrolinx and md warnings.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CustomVision

Commit:
redirection.json: fixed redirect error from previous commit.

Fixed toc structure mistake from previous commit.

Renamed "How-to guides" to "How-to".

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ContentModerator

Commit:
redirection.json: re-fixed redirect error from previous commit.

daxianji007 committed:
Update index.md

Fix hub page broken link

v-nagta committed:
update

v-nagta committed:
update

kadimitr committed:
Adding sample for use cases and extending instance management page.

v-nagta committed:
update

v-nagta committed:
User Story 1247764: SaaS App Tutorial: KnowledgeOwl

Commit:
Create how-to-ci-cd.md

v-nagta committed:
update

Commit:
IoT Edge CI/CD

Commit:
update toc

v-nagta committed:
update

Commit:
update image

BryanLa committed:
swap iot dev center with fundamentals

rolyon committed:
Deleted Get operations from activity log article

rolyon committed:
Deleted old screenshots

Commit:
Added intro sentence.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ContentModerator

parakhj committed:
Adding Azure AD article

parakhj committed:
Updating link to Azure AD article

parakhj committed:
Updating titles

v-deken committed:
Updated TOCs and landing pages for all Search services

v-stadam committed:
cleanup/sync landing pages

v-stadam committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into LandingPages

v-stadam committed:
fix YAML error

v-deken committed:
Fixed HTML escape code for C# links in TOC

Commit:
Changed "Review tool" to "Get started with Content Moderator".

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ContentModerator

Commit:
Change header

tdykstra committed:
ms.date

anirudhcavale committed:
Update IoT Hub

anirudhcavale committed:
More updates

JasonWHowell committed:
Update stream-analytics-define-outputs.md

JasonWHowell committed:
Update stream-analytics-define-inputs.md

ajlam committed:
Update pricing tiers

ajlam committed:
Update pg regions

parakhj committed:
Changing the name

ktoliver committed:
Merge pull request #38055 from yzhong94/ios-doc

Add how to develop for mobile platform

tfosmark committed:
Merge pull request #38275 from shae-hurst/sh-chunkdoc

LA large message handling with chunking

JasonWHowell committed:
Update stream-analytics-event-hub-consumer-groups.md

PRMerger10 committed:
Merge pull request #38991 from anirudhcavale/patch-6

Updating available metrics

PRMerger10 committed:
Merge pull request #39383 from ajlam/patch-1

Update regions PG and MySQL

JasonWHowell committed:
Merge branch 'master' of https://github.com/jasonwhowell/azure-docs-pr into sa-consumergroups

JasonWHowell committed:
New consumer group screenshot

PRMerger16 committed:
Merge pull request #39381 from JasonWHowell/patch-3

Update stream-analytics-define-inputs.md

PRMerger16 committed:
Merge pull request #39379 from JasonWHowell/patch-2

Update stream-analytics-define-outputs.md

ktoliver committed:
Merge pull request #39322 from rolyon/rolyon-logs-fix

[Azure RBAC] Logged operations fix, deleted old screenshots

dlepow committed:
Batch: quota increases

rmca14 committed:
Merge pull request #39344 from parakhj/oidcidp

Adding Azure AD article

Commit:
Merge pull request #39401 from dlepow/quota

Batch: quota increases

mmacy committed:
[ACR] build overview work

JasonWHowell committed:
Updating metadata and article formatting

ktoliver committed:
Merge pull request #39392 from JasonWHowell/sa-consumergroups

Adding screenshot and details on adding a new consumer group

JasonWHowell committed:
Update stream-analytics-test-query.md

JasonWHowell committed:
Update stream-analytics-test-query.md

v-deken committed:
Removed en-us locale from links in TOC

rmca14 committed:
Merge pull request #39268 from v-deken/LandingPage-RestOfSearch

Landing pages updated for Autosuggest, Custom Search, Entity Search, and Image Search

barclayn committed:
fixing the blue boxes

Commit:
Changed "How-to" to "How-to guides".

Commit:
Merge pull request #39417 from barclayn/blue-boxes

fixing the blue boxes

Commit:
Custom-Vision-Service/toc: changed "How-to" to "How-to guides".

Computer-vision/toc: changed "How-to guides" back to "How-to".
This was a mistake - change to be made in ComputerVision branch.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CustomVision

PRMerger10 committed:
Merge pull request #38841 from ramical/patch-1

Update sample to use AcquireTokenAsync

Commit:
toc: changed "How-to" to "How-to guides".

rolyon committed:
Added Azure Activity Log Analytics section

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-log-analytics

Commit:
Update images, update metadata and h2

gled4er committed:
Demonstrating how to use CustomStatus from DurableOrchestrationClientBase.

ktoliver committed:
Merge pull request #39091 from easyj2j/ContentModerator

Content Moderator

Commit:
TOC section on Develop-U-SQL programs organized to avoid randomly mixing language vs tools content

tfosmark committed:
Merge pull request #39288 from daxianji007/master

Update index.md

tfosmark committed:
Merge pull request #39406 from JasonWHowell/sa-testquery

Updating metadata and article formatting

Commit:
Update data-lake-analytics-overview.md

PRMerger18 committed:
Merge pull request #39452 from saveenr/patch-28

Update data-lake-analytics-overview.md

Commit:
removed pluralsite

v-shils committed:
Merge pull request #39153 from easyj2j/CustomVision

Custom Vision

v-shils committed:
Merge pull request #39295 from shizn/patch-1

IoT Edge CI/CD

v-stadam committed:
fixed metadata

v-stadam committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into LandingPages

v-shils committed:
Merge pull request #38979 from v-viinde/montageonline

USER STORY 1245548 SaaS App Tutorial: Montage Online

mattbriggs committed:
update ARM

v-shils committed:
Merge pull request #39133 from v-nagta/jamf

User Story 1246719: SaaS App Tutorial: Jamf Pro SAML Connector

v-shils committed:
Merge pull request #39136 from v-nagta/launchdarkly

User Story 1246722: SaaS App Tutorial: LaunchDarkly

v-shils committed:
Merge pull request #39209 from mattbriggs/mdb_20180426_apiprofile2

Update profile index

tdykstra committed:
Ross feedback

rolyon committed:
Incorporated feedback from Meir

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-log-analytics

rmca14 committed:
Merge pull request #39354 from v-stadam/LandingPages

Landing pages sync/clean-up

rjagiewich committed:
Merge pull request #39293 from v-nagta/knowledgeowl

User Story 1247764: SaaS App Tutorial: KnowledgeOwl

rmca14 committed:
Merge pull request #32951 from tdykstra/webjobs

New WebJobs SDK docs

tfosmark committed:
Merge pull request #39427 from easyj2j/FaceAPI

toc: changed "How-to" to "How-to guides".

ecfan committed:
Update art and steps for article

ecfan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into workflow-settings

tfosmark committed:
Merge pull request #39454 from davidmu1/b2clanding1

removed pluralsite

ecfan committed:
Address Acrolinx

ecfan committed:
Address Acrolinx

ecfan committed:
Restore folder

ecfan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into workflow-settings

Commit:
Update data-lake-analytics-u-sql-cognitive.md

SaeedAkhter-MSFT committed:
address feedback from auth sdk pm team

PRMerger20 committed:
Merge pull request #39473 from saveenr/patch-29

Update data-lake-analytics-u-sql-cognitive.md

kadimitr committed:
Fixing the spelling of discount.

giladmit committed:
clarified that Audit / SQLSecurityAuditEvents aren't available

Commit:
Merge pull request #39474 from giladmit/patch-2

clarified that Audit / SQLSecurityAuditEvents aren't available

Commit:
Update to MVC.

toc:
Renamed "Video Indexer overview" to "What is Video Indexer"
Renamed "REST" to "Video Indexer API"
Added stock Resources items.

Overview:
Updated metadata.
Changed "Video Indexer" to "What is Video Indexer?"
Fixed some acrolinx warnings.

Commit:
Changed "How to" to "How to guides".

mmacy committed:
[ACR] finish up OS & framework patch section

mmacy committed:
[ACR] typo fix

Commit:
Update to MVC.

toc:
Made Overview a dropdown with 1 item: "What is Emotion API"
Renamed Quickstarts to "Try Emotion API" and moved to How-to.
Renamed Tutorials to "First Emotion API apps" and moved to How-to.
Changed "How-to" to "How-to guides".
Renamed Reference/Emotion Reference to Emotion API.
Added stock Resources items; deleted Samples as there are none.

Overview: Changed "Emotion API" to "What is Emotion API?".
Changed Important message.

neilpeterson committed:
updated template to use parameter for container group name

Commit:
Reference: delete FAQ.

Overview: updated metadata.

Commit:
Merge pull request #39481 from neilpeterson/aci-issue-7778

[ACI] fixes MicrosoftDocs/azure-docs#7778

mmacy committed:
[ACR] minor copy edit

mmacy committed:
[ACR] add Preview note

Commit:
Update data-lake-analytics-manage-use-powershell.md

PRMerger14 committed:
Merge pull request #39483 from saveenr/patch-30

Update data-lake-analytics-manage-use-powershell.md

Duncanma committed:
Make code sample have a language

ktoliver committed:
Merge pull request #38822 from gled4er/kaniodimitrov/durable-functions-custom-orchestration-status

Adding content for Custom orchestration status

ktoliver committed:
Merge pull request #38906 from mmacy/acr-build-overview

[ACR] new: ACR Build overview

SaeedAkhter-MSFT committed:
Acrolinx suggestions

ktoliver committed:
Merge pull request #37879 from SaeedAkhter-MSFT/saeeda-auth-scenarios

388665 review auth scenarios

ecfan committed:
Add content about unlinking

tynevi committed:
Create index.yml

tynevi committed:
Delete index.yml

tynevi committed:
Merge pull request #39489 from MicrosoftDocs/text-analytics-problem

Create index.yml in correct text-analytics folder

tynevi committed:
Delete edit-queries.png

tynevi committed:
Adding edit-queries.png to lowercase path

tynevi committed:
Merge pull request #39490 from MicrosoftDocs/text-analytics-problem

Text analytics problem

raiye committed:
Update virtual-machines-common-how-to-enable-write-accelerator.md

Added GA-relevant information

rolyon committed:
Fixed small typo

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-log-analytics

Sogup committed:
Update backup-azure-vms-first-look-arm.md

Commit:
Merge pull request #39503 from Sogup/patch-69

Update backup-azure-vms-first-look-arm.md

Commit:
Merge pull request #39172 from jimdial/patch-10

Removed region constraints

Commit:
Updated tag information.

PRMerger14 committed:
Merge pull request #39512 from jimdial/patch-11

Updated tag information.

PRMerger12 committed:
Merge pull request #39500 from raiye/patch-2

Update virtual-machines-common-how-to-enable-write-accelerator.md

itechedit committed:
Merge pull request #39429 from rolyon/rolyon-log-analytics

[Azure RBAC] Azure Activity Log Analytics

itechedit committed:
Merge pull request #39450 from saveenr/patch-27

TOC section on Develop-U-SQL programs organized to avoid randomly mix…

rjagiewich committed:
Merge pull request #39303 from BryanLa/bryanla-hub-page-iot-updates

Hub page IoT updates - Swap iot dev center with fundamentals

itechedit committed:
Merge pull request #39480 from easyj2j/VideoIndexer

Video Indexer

rjagiewich committed:
Merge pull request #39467 from ecfan/workflow-settings

Update UX and steps per GitHub feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #39534 from MicrosoftDocs/FromPublicRepo

From public repo

v-shils committed:
Merge pull request #39482 from easyj2j/EmotionAPI

EmotionAPI-Update to MVC.
